vendor/assets/javascripts/jquery.validationEngine-es.js

```diff
@@ -1,139 +1,143 @@
+          "alertText": "* Direccion IP inválida"
+        },
+        "url": {
+          "regex": /^(https?|ftp):\/\/(((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:)*@)?(((\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.(\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.(\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.(\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5]))|((([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.)+(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.?)(:\d*)?)(\/((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)+(\/(([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)*)*)?)?(\?((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)|[\uE000-\uF8FF]|\/|\?)*)?(\#((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)|\/|\?)*)?$/i,
+          "alertText": "* URL Inválida"
+        },
+        "onlyNumberSp": {
+          "regex": /^[0-9\ ]+$/,
+          "alertText": "* Sólo números"
+        },
+        "onlyLetterSp": {
+          "regex": /^[a-zA-Z\ \']+$/,
+          "alertText": "* Sólo letras"
+        },
+        "onlyLetterAccentSp":{
+          "regex": /^[a-z\u00C0-\u017F\ ]+$/i,
+          "alertText": "* Sólo letras"
+        },
+        "onlyLetterNumber": {
+          "regex": /^[0-9a-zA-Z]+$/,
+          "alertText": "* No se permiten caracteres especiales"
+        },
+        // --- CUSTOM RULES -- Those are specific to the demos, they can be removed or changed to your likings
+        "ajaxUserCall": {
+          "url": "ajaxValidateFieldUser",
+          // you may want to pass extra data on the ajax call
+          "extraData": "name=eric",
+          "alertTextLoad": "* Cargando, espere por favor",
+          "alertText": "* Este nombre de usuario ya se encuentra usado"
+        },
+        "ajaxNameCall": {
+          // remote json service location
+          "url": "ajaxValidateFieldName",
+          // error
+          "alertText": "* Este nombre ya se encuentra usado",
+          // if you provide an "alertTextOk", it will show as a green prompt when the field validates
+          "alertTextOk": "* Este nombre está disponible",
+          // speaks by itself
+          "alertTextLoad": "* Cargando, espere por favor"
+        },
+        "validate2fields": {
+          "alertText": "* Por favor entrar HELLO"
+        },
+        "validate_encoding": {
+          "regex": /[^\u0000-\u00ff]/g;
```

Comment:
This regular expression is not valid because is not validated all characters of the description
![captura de pantalla de 2016-07-25 11 06 50](https://cloud.githubusercontent.com/assets/15264501/17096078/f4c9087a-5257-11e6-8538-10db8c8564b6.png)